src/app/characters/character-list/character-list.component.ts

```diff
@@ -37,20 +40,42 @@ export class CharacterListComponent implements OnInit {
 
   trackByCharacters(_: number, character: Character) { return character.id; }
 
+  private showWarning() {
+    this.matSnackBarRef = this.snackbar.open(
+      'There are more results of your search that are not currently displayed. Please try to refine your search criteria.',
+      'Dismiss',
+      { duration: 20000 },
+    );
+  }
+
+  private dismissWarning() {
```

Comment:
I think it would be more clear if we could use the null coalescing operator to dismiss the snackbar and get rid of this method such as `this.matSnackBarRef?.dismiss();`

Reply:
Done. Great idea.